jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -48,101 +48,101 @@
     - kubernetes-soak-gce-deploy:
         blocker: ci-kubernetes-soak-gce-test
         job-name: ci-kubernetes-soak-gce-deploy
-        json: 0
+        json: 1
```

Comment:
Time to kill this json flag :-) Now we can pass `json=1` in line 9 and eliminate the unit-test check